communitiesuk/funding-service-design-assessment-store
# Fs 4739 cof25 configs

https://mhclgdigital.atlassian.net/browse/FS-4739


### Change description
- Mapping added for COF25
- Tests added to verify newly added configs
- Inserted cof25 round in to assessment round
- cof25 submitted application seed data added




### How to test
- Make sure you have cof25 related roles assigned
- Make and send an application for cof25
- Login to assessment, validate cof25 configs exist and you can see the received application
- Validate the export options in COF25 work as expected

### Screenshots of UI changes (if applicable)
![Screenshot 2024-10-30 at 14 22 44](https://github.com/user-attachments/assets/c2894de3-6829-4ffb-a5d0-5158e976c52d)


![Screenshot 2024-10-30 at 14 23 08](https://github.com/user-attachments/assets/3c88f824-4481-4485-9bdc-5da9f06c74f6)


![Screenshot 2024-10-30 at 14 23 25](https://github.com/user-attachments/assets/da3cb8a3-68e7-4698-9ee4-a70b2448109c)


![Screenshot 2024-10-30 at 14 23 40](https://github.com/user-attachments/assets/c9384a45-89fc-429b-bc9a-d6f42d2d8720)
